server/paymentProviders/utils.ts

```diff
@@ -174,10 +174,12 @@ export const persistTransaction = async (virtualCard, transaction) => {
       });
       if (originalCreditTransaction?.amount === amount) {
         if (!originalCreditTransaction.RefundTransactionId) {
-          await createRefundTransaction(originalCreditTransaction, 0, {
-            refundTransactionId: transactionId,
-            transaction,
-          });
+          await createRefundTransaction(
+            originalCreditTransaction,
+            0,
+            { refundTransactionId: transactionId, transaction },
+            null,
```

Comment:
`user` param does not have a default so we have to provide it. Since it's unclear who the user should be here, we're explicitly passing null.